scripts/backend.py

```diff
@@ -14,10 +14,23 @@
 """
 
 from docopt import docopt
-from flask import Flask, make_response, redirect, request, send_file
+from flask import Flask, make_response, redirect, request, send_file, render_template
 from mpd import MPDClient
 import json, os, random, time
 
+### all of these are added dependencies by puss, stdlib
+# import threading  # uncomment this if you want stream to start after current song ends
```

Comment:
I think the "start after current song ends" code can be removed, as in practice people will never start their stream at precisely the right second.